### PR TITLE
update and add FAQ content

### PR DESCRIPTION
**Pull request by mindovermiles262**

### CODE REVIEW

**Comment by KevinMulhern** [comment]
Hey @mindovermiles262 this is looking really good 😄 thanks for putting it together.
I think we should keep the sections about the "how much it costs" and "how will be sustainable". Just so we can communicate that the site is completley open source and updated regularly by volunteer maintainers and if they want to show thier appreciation for the site they can contribute.

Austin wrote some new content for those parts, they were merged in recently so thats where the conflicts for this PR are coming from. Fixing the merge conflict to include his stuff will make this PR ready to merge.

Thanks again for contributing and sorry it took me so long to review this.

File: app/views/static_pages/faq.html.erb [hunk 1]
```diff
@@ -98,6 +97,20 @@
...
+   		You don't! The Odin Project is open-source and completely free.
+    </p>
+    <p>
+    	If you insist on compensation, you can donate your time to help those just starting The Odin Project or create a new feature for the site.
+    </p>
+
+
+	<h2 class="faq-question">How will The Odin Project become sustainable?</h2>
+    <p>
+ 		The Odin Project's business model is educating students.  Core content will always be free, but we'd also like to give students who want more structure and/or help on their journeys access to those services, which would be at a premium. If you want to help please <%= link_to "reach out", contact_path %>.
```
Comment (codyloyd) [inline app/views/static_pages/faq.html.erb]: what is this referring to?

File: app/views/static_pages/faq.html.erb [hunk 2]
```diff
@@ -5,124 +5,127 @@
...
     </p>
     <p> 
-      We've essentially distilled down what you most need to learn to hit that employable level, but it's still a healthy dose of learning.  Depending how fast you are, it will still take roughly 1000 hours of work to hit that sweet spot.  If you're naturally more technical or come from a technical background, it may be a bit faster.  If you're less technical or brand new to all this, it will take longer.  Don't despair!  When you think about it, that's pretty much the same learning curve you had to climb to learn anything worthwhile so far in life.  
-    </p>
-    <p>
-      If you're doing it 40 hours a week, that's about 6 months.  If you're super intense, you can do it in 3.  If you're part-timing or less, you'll need to space it out over about a year.  That's just the reality of it.
+      If you work on The Odin Project full-time 40 hours a week, it’ll take about 6 months. Part time learners can expect to take 12 months to finish.
```
Comment (codyloyd) [inline app/views/static_pages/faq.html.erb]: maybe I'm just picky.. but I like the original wording better

**Comment by mindovermiles262** [comment]
Changed wording of "How long does it take?" back to original.  I changed a few minor words to keep voice consistent

**Comment by codyloyd** [comment]
LGTM -> let's see what @KevinMulhern or @arun1595 have to say about it

**Comment by KevinMulhern** [comment]
Lets merge it 😄 the data structure in the redesign is different but I'll sort that out.

Thanks for putting this together @mindovermiles262

**Pull request merged**